binzx/otomi

```diff
@@ -176,7 +182,7 @@ check_update() {
   fi
 }
 
-[ -z $DONT_CHECK_UPDATE ] && [ -z $IN_DOCKER ] && silent echo "Checking for updates" && check_update
+[ -z $DONT_CHECK_UPDATE ] && [ "${IN_DOCKER,,}" != "false" ] && silent echo "Checking for updates" && check_update
```

Comment:
Since we run Otomi without docker then `IN_DOCKER` check may not be applicable anymore.
However `DONT_CHECK_UPDATE` should be set for tekton.

Reply:
I agree let's do it as a part of a new ticket.